test/CMakeLists.txt

```diff
@@ -464,6 +479,7 @@ if(LINUX)
             providers/level_zero_helpers.cpp
             LIBS
             ze_loader
+            disjoint_pool
```

Comment:
Why does it belong to this PR? What does it have in common?

Reply:
I have to use Disjoint Pool in GPU tests